src/main/java/helpbuddy/command/Command.java

```diff
@@ -0,0 +1,40 @@
+package helpbuddy.command;
```

Comment:
I think it's really great that you implemented the stretch goal of splitting each of your commands into another class.